dagobah/core/dagobah.py

```diff
@@ -100,9 +100,9 @@ def _add_job_from_spec(self, job_json, use_job_id=True):
             job.schedule(job_json['cron_schedule'])
 
         for task in job_json.get('tasks', []):
-            # If it is a jobtask, it will have a job_id
-            if task.get('job_id'):
-                self.add_jobtask_to_job(job, task['job_id'], str(task['name']))
```

Comment:
if the name key in `task['name']` does't exist it will raise a `keyError` exception. I think you should handle the possibility. Sam for `job_id`. perhaps using `get`?